envoy/datadog_checks/envoy/envoy.py

```diff
@@ -67,7 +67,10 @@ def check(self, instance):
             except ValueError:
                 continue
 
-            value = int(value)
+            try:
+                value = int(value)
+            except (ValueError, TypeError):
+                continue
```

Comment:
If we run into this case, would we be dropping a metric? If so, what do you think about adding a debug log line just mentioning that we won't/can't submit it? I'm not sure how noisy that would be but may be helpful in troubleshooting any potential cases later.